index.js

```diff
@@ -156,6 +156,12 @@ Hbs.prototype.createRenderer = function() {
     var tplPath = path.join(hbs.viewPath, tpl + hbs.extname),
       template, rawTemplate, layoutTemplate;
 
+    // if the template passed has any non-word characters leading,
+    // we're probably trying to pass an absolute path, so roll with that.
+    if (!fs.existsSync(tplPath) && !/^\w/.test(tpl)) {
```

Comment:
This won't work for windows style absolute paths `C:\Path`. Combining the regex test with an exists check is a bit odd as well. I think it might be better to refactor the `tplPath` generation into a function which inspects the received `tpl` for leading `/` or `/^[a-zA-Z]:\//`. I guess we can check if `path.isAbsolute` is defined for node 12.x series and then fallback to such a regex implementation for older versions.